app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -47,20 +53,47 @@ const NoteForm = ({ onClose, refetch, note, isEdit }) => {
               className="w-full flex-grow-0"
               label="Description"
               name="description"
-              rows={8}
+              rows={1}
+            />
+            <Select
+              isClearable
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Assigned Contact"
+              name="role"
+              options={buildSelectOption(ROLE_DATA)}
+              placeholder="Select Role"
+            />
+            <Select
+              isClearable
+              isMulti
+              isSearchable
+              required
+              className="w-full flex-grow-0"
+              label="Tags"
+              name="tags"
+              options={buildSelectOption(TAGS_DATA)}
+              placeholder="Select Tags"
             />
           </Pane.Body>
           <Pane.Footer>
             <Button
-              className="mr-3"
+              className="mr-3 rounded-sm bg-gray-800 hover:bg-gray-600"
               disabled={isSubmitting}
+              icon={Check}
               label={isEdit ? "Update" : "Save changes"}
               loading={isSubmitting}
               style="primary"
               type="submit"
               onClick={() => setSubmitted(true)}
             />
-            <Button label="Cancel" style="text" onClick={onClose} />
+            <Button
+              className="rounded-sm"
+              label="Cancel"
+              style="text"
+              onClick={onClose}
```

Comment:
Go through the Slack thread.

Reply:
Thanks for pointing out this mistake. I added the `type` as `reset` in the code as per the [link](https://academy.bigbinary.com/learn-react/pages/formik-forms#:~:text=Formik%20cancel%20button%20type).